projects/igniteui-angular-elements/tsconfig.app.json

```diff
@@ -0,0 +1,17 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
Wouldn't that be fun, considering it got generated like that initially :)
Yeah, technically this could be considered `jsonc` I suppose, since both tsconfig and angular.json do fine with comments